cmd/config.go

```diff
@@ -164,7 +164,6 @@ type Config struct {
 	PA PAConfig
 
 	Common struct {
-		BaseURL string
```

Comment:
Unfortunately we can't delete this from Common yet because that's still where the config is set in prod. The boulder-config/boulder-config-next dichotomy does not yet reflect that because it predates the config split. If you want to update the configs to reflect that reality I'd appreciate it!